r/tools/nixlibs.R

```diff
@@ -268,7 +272,7 @@ get_macos_openssl_dir <- function() {
       openssl_root_dir <- "/usr/local"
     }
   }
-  return(openssl_root_dir)
+  openssl_root_dir
```

Comment:
This wasn't technically necessary, but I wanted to clean up some of the inconsistencies in this file and follow our general style of no `return` if it's the last value in a function.